tooling/cli/src/migrate/migrations/v1/config.rs

```diff
@@ -114,6 +114,10 @@ fn migrate_config(config: &mut Value) -> Result<MigratedConfig> {
       process_updater(tauri_config, &mut plugins)?;
     }
 
+    if plugins.contains_key("updater") {
```

Comment:
shouldn't this be `if let Some(updater) = tauri_config.remove("updater") {` (moved into the above block)?

Reply:
I didn't want to add the migrated argument (lazy) but it's indeed cleaner that way.. pushed

Reply:
ah tbh i just missed the process_updater line above that (its comment is saying `cli` btw) so i thought the code is actually incorrect.